.github/workflows/frontend-shared.yml

```diff
@@ -0,0 +1,44 @@
+name: frontend-shared package
+on:
+  release:
+    types:
+      - published
+jobs:
+  update-frontend-shared-version:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    # Additionally fetch master to be able to diff for changes in the frontend-shared/ folder
+    - name: Fetch origin
+      run: git fetch origin master:master
+    # Setup .npmrc file to publish to npm
+    - name: Setup Nodejs
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - name: Install client
+      run: yarn install --frozen-lockfile
+    - name: Get new version
+      uses: ./.github/actions/frontend-shared-version
+      id: version
+      with:
+        shared-package: '@hypothesis/frontend-shared'
+        parent-package: 'hypothesis'
+    - name: Set up frontend-shared
+      run: yarn setup-frontend-shared
+      # If updated_version is null, skip this step
+      if: ${{ steps.version.outputs.updated_version }}
```

Comment:
I wonder if there is a way to exit early if `updated_version` is null rather than repeating the `if:` on each subsequent step?

Reply:
I thought the same thing, I looked into it for about 5mins, but did not find much. I'll do some more searches about early exit.